napari/_event_loop.py

```diff
@@ -1,5 +1,5 @@
 try:
-    from napari._qt.qt_event_loop import gui_qt, run
+    from napari._qt.qt_event_loop import run
```

Comment:
Is this intentionally in this PR?

Reply:
Nope. I cherry-picked 1 too many commits.